Gemfile.lock

```diff
@@ -311,7 +311,7 @@ GEM
       rails (>= 3.2.16)
     xpath (3.2.0)
       nokogiri (~> 1.8)
-    zeitwerk (2.4.2)
+    zeitwerk (2.5.1)
```

Comment:
Are these upgrades related at all?

Reply:
They should be `webpacker` dependency updates as they were updated as a result of `bundle update webpacker`. We had updated the Gemfile directly when bumping just the Ruby side of webpacker previously, see [PR](https://github.com/artsy/horizon/commit/92db616ffb7f7f90686df18327a7df24b726e37c), which is why we aren't seeing a direct `webpacker` bump in this PR, but just its dependencies.

For this update, I followed the full upgrade instructions from `webpacker` [here](https://github.com/rails/webpacker#upgrading).